src/test/java/guitests/StartupBoardLauncherTest.java

```diff
@@ -0,0 +1,54 @@
+package guitests;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.loadui.testfx.utils.FXTestUtils;
+import prefs.Preferences;
+import ui.BoardAutoCreator;
+import ui.TestController;
+import ui.UI;
+import ui.issuepanel.PanelControl;
+import static guitests.BoardAutoCreatorTest.verifySampleBoard;
+
+import java.util.Optional;
+
+import static org.junit.Assert.assertEquals;
+import static ui.BoardAutoCreator.SAMPLE_BOARD;
+
+public class StartupBoardLauncherTest extends UITest{
+
+    private PanelControl panelControl;
+    private Preferences testPref;
+
+    @Override
+    public void launchApp(){
+        FXTestUtils.launchApp(TestUI.class, "--startupboard=true");
+    }
+
+    @Before
+    public void init(){
+        UI ui = TestController.getUI();
+        panelControl = ui.getPanelControl();
+        testPref = UI.prefs;
+    }
+
+    @Test
+    public void testSampleBoardOnLaunch(){
+        //First we login and then logout for first time users.
```

Comment:
This comment is redundant. Remember, we should comment about WHY, now HOW. The HOW should be obvious from the code.